Re #72: Add an option in settings to set icon style

smitgd opened the pull request:
This allows selection between KDE theme weather icons or the new font
based icons on the main plasmoid UI (both the large "condition" icon
and the smaller icons for the 10 day forecast.
Note: The new font based icons are _always_ used when installed to panel
and KDE theme weather icons still are not selectable there due to
readability issues.
    modified:   plasmoid/contents/config/main.xml
    modified:   plasmoid/contents/ui/ConfigGeneral.qml
    modified:   plasmoid/contents/ui/Forecast.qml
    modified:   plasmoid/contents/ui/ForecastDelegate.qml
    modified:   plasmoid/contents/ui/Weather.qml
    modified:   plasmoid/contents/ui/Yahoo.qml

librehat commented on plasmoid/contents/ui/ConfigGeneral.qml:
```diff
@@ -61,5 +62,16 @@ Item {
         }
+
+        ColumnLayout {
+            Label {
```
Why putting an empty label here?

librehat commented on plasmoid/contents/ui/ForecastDelegate.qml:
```diff
@@ -23,11 +24,28 @@ Column {
+        visible: !useWxFonts
         source: icon
-        width: theme.smallMediumIconSize
+        width: useWxFonts ? 0 : theme.smallMediumIconSize
```
Do we need to set width here? When `useWxFonts` is true, this item is not even visible.

librehat commented on plasmoid/contents/ui/Weather.qml:
```diff
@@ -14,13 +14,16 @@ import QtQuick.Controls 1.2
     Layout.minimumWidth: units.gridUnit * 18
     Layout.minimumHeight: units.gridUnit * 20
     clip: true
+    property bool useWxFonts: plasmoid.configuration.useWxFonts 
```
or alias?

smitgd commented:
I put the "empty" labels in a while back to improve spacing/readability of configuration items. There may be a better way, not sure. I will look into it.

Made the width and height zero when not visible to make sure no blank space is present when not visible. Actually, never tried it without setting height/width to zero so I will try it and see what happens. I think you are right, probably don't need it.

Maybe "alias" is a better way so I will check. (I don't know anything about "alias" right now, so will learn.)

smitgd committed:
Re #72: Add an option in settings to set icon style
This allows selection between KDE theme weather icons or the new font
based icons on the main plasmoid UI (both the large "condition" icon
and the smaller icons for the 10 day forecast.
Note: The new font based icons are *always* used when installed to panel
and KDE theme weather icons still are not selectable there due to
readability issues. The pop-up "tool-tip" that appears on mouse over
on the panel now also show the selected KDE theme icon or the font based
symbol "icon".
This also removes the unneeded blank Labels from ConfigGeneral.qml and
places Labels and TextFields on same line for somewhat improved
readability.
	modified:   plasmoid/contents/config/main.xml
	modified:   plasmoid/contents/ui/ConfigGeneral.qml
	modified:   plasmoid/contents/ui/Forecast.qml
	modified:   plasmoid/contents/ui/ForecastDelegate.qml
	modified:   plasmoid/contents/ui/Weather.qml
	modified:   plasmoid/contents/ui/Yahoo.qml

smitgd commented:
Updated this pull req. to address the issues above. Couldn't get alias to plasmoid.configuration.useWxFonts to work since got error that "plasmoid" is not an "id". So just used plasmoid.configuration.useWxFonts directly instead of using alias or defining a new variable..
Also, noticed on the tool-tip that appears on mouse over of the panel installation that it showed the KDE theme icon always so fixed that too. See updated commit message for more details.

librehat commented:
Seems good. Thank you!
